rns_shell/platform/graphics/x11/WindowX11.h

```diff
@@ -10,6 +10,7 @@
 
 #include "ReactSkia/utils/RnsUtils.h"
 
+#include "ReactSkia/sdk/NotificationCenter.h"
```

Comment:
Just add a comment saying why this is kept on top. To avoid the **None** conflict with X11